pages/common/ffmpeg.md

```diff
@@ -18,6 +18,10 @@
 
 `ffmpeg -ss {{mm:ss}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
 
+- Trim a video from a given time (ss) to (t) seconds from that time. Just omit the -t flag to only trim everything before (ss) seconds:
```

Comment:
How about "Trim a video from a given start time mm:ss to a duration of mm2:ss2 from that time. (Omit the -t flag to trim till the end):" ? And change the example duration to `mm2:ss2`

Reply:
Yes, It's more fluent! Thanks for the suggestion! :)